zeus/reports.py

```diff
@@ -374,7 +374,7 @@ def csv_from_score_polls(election, polls, lang, outfile=None):
         try:
             outfile.seek(0)
             return outfile.read()
-        except:
+        except ValueError:
```

Comment:
Same as above. Maybe these blocks (`seek(0)` and `read`) are not actually necessary.